tests/test_questions.py

```diff
@@ -64,25 +64,40 @@ def test_stores_index_path(self) -> None:
         assert botlr.body_parts == ["shuffled", "off", 10, "mortal", "coils"]
 
     def test_digs_into_json(self, APITester: Actor) -> None:
-        test_json = {"plays": [{"name": "Hamlet"}]}
+        test_json = {"plays": [{"name": "Hamlet: A Horror Story"}]}
         fake_response = mock.Mock()
         fake_response.json.return_value = test_json
         mocked_mar = APITester.ability_to(MakeAPIRequests)
         mocked_mar.responses = [fake_response]
 
-        botlr = BodyOfTheLastResponse()["plays"][0]["name"]
+        botlr = BodyOfTheLastResponse()["plays"][0]["name"][:6]
```

Comment:
I'm probably missing it but; why add the extra portion of the name?

Reply:
I wanted to test the slicing as well. This test previously got indexing by string and number, but no slicing.